riotfile.py

```diff
@@ -298,6 +322,19 @@ def select_pys(min_version=min(SUPPORTED_PYTHON_VERSIONS), max_version=max(SUPPO
                 ),
             ],
         ),
+        Venv(
+            name="sqlalchemy",
+            command="pytest {cmdargs} tests/contrib/sqlalchemy",
+            venvs = [
+                Venv(
+                    pys=select_pys(),
+                    pkgs={
+                        "sqlalchemy": ["~=1.0.0", "~=1.1.0", "~=1.2.0", "~=1.3.0", latest],
+                        "psycopg2": ["~=2.8.0"],
```

Comment:
I feel like we should install `SQLAlchemy[postgresql]` and not psycopg2 directly here.
That should be changed in a different PR anyway.